youtube_dlc/options.py

```diff
@@ -654,6 +654,11 @@ def _dict_from_multiple_values_options_callback(
         help=(
             'Use the mpegts container for HLS videos, allowing to play the '
             'video while downloading (some players may not be able to play it)'))
+    downloader.add_option(
+        '--hls-ignore-discontinuity',
+        dest='hls_ignore_discontinuity', action='store_true', default=None,
+        help='Ignore media discontinuity on HLS playlists (do not split ad breaks)'
+    )
```

Comment:
```suggestion
    downloader.add_option(
        '--hls-split-discontinuity',
        dest='hls_ignore_discontinuity', action='store_false', default=True,
        help='Split discontinuity on HLS playlists to different formats'
    )
    downloader.add_option(
        '--no-hls-split-discontinuity',
        dest='hls_ignore_discontinuity', action='store_true',
        help='Concatenate media discontinuity on HLS playlists (default)'
    )
```

Feel free to change the actual switch name and description.